### ISSUE TITLE
git-download.sh script broken on GHE 2.12.x

### ISSUE THREAD

**Issue by rashamalek** [Autodesk/hubble#98]
Due to a log format (order) change on github-audit.log.* on GHE 2.12.x (we tested that on GHE 2.12.3), git-download.sh is broken.

Here is a proposal to fix it using awk:

Current one which works on =<2.11.x
```
eval "$CAT_LOG_FILE" |
    perl -ne 'print if s/.*"program":"upload-pack".*"repo_name":"([^"]+).*"user_login":"([^"]+).*"cloning":([^,]+).*"uploaded_bytes":([^ ]+).*/\1\t\2\t\3\t\4/' |
    sort |
    perl -ne '$S{$1} += $2 and $C{$1} += 1 if (/^(.+)\t(\d+)$/);END{printf("%s\t%i\t%i\n",$_,$C{$_},$S{$_}) for ( keys %S );}' |
    sort -rn -k5,5
```

For GHE >=2.12
```
eval "$CAT_LOG_FILE" |
	perl -ne 'print if s/.*"cloning":([^,]+).*"program":"upload-pack".*"repo_name":"([^"]+).*"uploaded_bytes":([^,]+).*"user_login":"([^"]+).*/\1\t\2\t\3\t\4/'
	| awk '{print $2"\t"$4"\t"$1"\t"$3}'
	| sort
	| perl -ne '$S{$1} += $2 and $C{$1} += 1 if (/^(.+)\t(\d+)$/);END{printf("%s\t%i\t%i\n",$_,$C{$_},$S{$_}) for ( keys %S );}'
	| sort -rn -k5,5
```

**Comment by larsxschneider** [comment]
Thanks a lot for the heads-up and the proposed fix 👍 
A minor improvement: you can change the order of the fields in the perl script line above at the end!

When you make the commit, can you add an example line for github-audit.log in the commit message to explain the change?

**Comment by pluehne** [comment]
Fixed by #107 by @rashamalek.
